### ISSUE TITLE
Resource intake: False negative rates on tests

### ISSUE THREAD

**Issue by blanchardjeremy** [microcovid/microcovid#316]
[This is the only data](https://www.hopkinsmedicine.org/news/newsroom/news-releases/covid-19-test-that-relies-on-viral-genetic-material-gives-false-negative-results-if-used-too-early-in-those-infected) I've seen on false negatives. Have y'all seen anything else?

- 4 days after infection: 67% false negative
- 8 days after infection: 20% false negative (3 days after symptoms)

_Originally posted by @blanchardjeremy in https://github.com/microcovid/microcovid/issues/215#issuecomment-687231614_

Two articles to consider doing an intake on:

* https://www.nejm.org/doi/full/10.1056/NEJMp2015897
* https://www.medrxiv.org/content/10.1101/2020.04.16.20066787v2

**Comment by cstigler** [comment]
Something that the linked article doesn't seem to address (or I missed it!) is the difference between testing _positive_, being _symptomatic_, and being _contagious_. My understanding is that a person can test positive based on an extremely small amount of virus, sometimes well before they're actually contagious. (see: [this NYTimes article about the extremely aggressive cycle thresholds](https://www.nytimes.com/2020/08/29/health/coronavirus-testing.html). In this case, what the MicroCOVID calculation really cares about is whether they're _currently_ contagious (at time of event), not whether they're infected, so it seems like the distinctions matter.

As an example scenario, a person who was infected on Saturday, tested on Sunday, hangs out with a friend on Monday, and starts being contagious on Tuesday would count as a "false negative" (if I'm understanding correctly) since they were infected at the time of the test. But the test result was still useful, because it correctly indicated that the the person would not transmit the virus to their friend.

Does that logic make sense to you? I might be totally off-base, but if that general line of reasoning is right, you'd almost need to convert the standard reported false negative rates into a probability table: if person X tested negative on Day 0, what is the probability they're _contagious_ on Day 1, Day 2, Day 3, Day 4... whether they _will_ develop COVID-19 isn't really the point for MicroCOVID, it's whether they _are currently_ contagious.

Following and curious to hear any thoughts because our own 11-person SF group house is trying to figure out how to weight testing :)

**Comment by beshaya** [comment]
Hi @cstigler,

I see two parts to your question: "how should we interpret timing of test results interact with timing of contagiousness" and "how should testing interact with microCOVID". I'll try to address these separately.

*How does testing interact with contagiousness?*
> As an example scenario, a person who was infected on Saturday, tested on Sunday, hangs out with a friend on Monday, and starts being contagious on Tuesday would count as a "false negative" (if I'm understanding correctly) since they were infected at the time of the test. But the test result was still useful, because it correctly indicated that the the person would not transmit the virus to their friend.
whether they will develop COVID-19 isn't really the point for MicroCOVID, it's whether they are currently contagious.

Yes, the chance that they are currently contagious is what matters for transmission risk, which is what microCOVID attempts to model. Unfortunately, this isn't how the timing of testing vs contagiousness works to my knowledge.

From https://www.medrxiv.org/content/10.1101/2020.04.25.20079889v1.full.pdf:
>Viral load studies suggest peak viral load occurs close to symptom onset (potentially, -1 to 7 days of [symptom] onset),

And:
> The pre symptomatic infectious phase (T3) is quite narrowly defined to a mean of approximately 2 days (range: <1-4)

From https://www.nature.com/articles/s41591-020-1046-6:
> a study examining 468 COVID-19 cases in China found that 12.6% of transmission occurred prior to symptom onset

Meanwhile testing is most accurate 3 days after symptom onset.

Put together, people start being contagious 4-5 days before the ideal time to take a test. This is counter-intuitive; I also originally guessed that testing would pick up on contagious individuals. But the same results are echoed everywhere, people are contagious before they reliably test positive.

So what does that mean for policy around testing?

If you test positive, you should inform everyone you have seen to at least 8 days prior to your test (I'd say 2 weeks to be safe). They should isolate until they can be tested. They should also inform everyone they have been in contact with starting the day they were in contact with you, and _those_ people should isolate - on the low end it only takes 2 days from exposure to become contagious.

If you test negative, it _doesn't_ mean you're not contagious. A test at the ideal time still has a 20% false negative rate, so it updates the chance you have covid by x0.2. You can combine this with waiting 10 days (90% of people who will develop symptoms do so within 10 days of exposure) or 14 days (99%) to x0.02 or 0.002 respectively.

*How does this interact with microCovid?*
The microCovid spreadsheet is designed for tracking and managing _small_ risks. (At the scale of 1-in-a-million 🙂) There are lots of second order effects (like the table you mention, or the 2-10 day window) that we just don't bother with because 1% of a millionth is trivial. If someone is exposed to a very high risk, you'd have to do some of the calculations manually. For instance, if your housemate has confirmed covid, you are now at 300,000 microCovid. Waiting 10 days only reduces this to 30,000. A properly timed negative test further reduces it to 6,000. Waiting another 4 days without symptoms gets you to 600. The spreadsheet is not smart enough to deal with this, as it simply forgives all risk after 10 days.

That said, if you're using testing to clear what we call "high risk activities" like air travel, testing is much more appealing - If you rack up 1000microCovid, waiting 10 days brings you to 100, but 10 days with a negative test brings you to 20.

If you stay within your budget, testing is generally unnecessary - 200microCovid becomes 20 after 10 days or 2 after 14 with no symptoms. Treating it as 0 after 10 days introduces at most a 10% error to your annual risk. Given that test are most accurate 8 days after exposure and take 2 days to get results back, you're unlikely to get results before lack of symptoms suggests that you are cleared.

The result is that we rarely get tested unless doing something that is more risky than the normal weekly budget (mostly flying to see family).

Hope this helps!

**Comment by blanchardjeremy** [comment]
@beshaya  this is a brilliant reply. I'm excited for your idea of starting an "ask microCOVID" section of the site where we can currate stuff like this.

**Comment by cstigler** [comment]
Wow, @beshaya, thanks so much for the detailed and thoughtful response - and especially for the links/quotes from those papers which were super informative!

Just to clarify (since I didn't above), the cases I'm looking into are actually _not_ the "when can I stop isolating if I get COVID" question (which should happen only ~once, hopefully, and being extra cautious isn't a big issue), but more: 1) does it reduce our risk to ask our friends to get negative test results before high-risk events, and 2) if one of us does a high-risk activity, to what degree does masking in the house / getting tested afterward alleviate that risk?

I wrote up our thinking on the first topic (the value of pre-event testing) [here](https://github.com/microcovid/microcovid/issues/215#issuecomment-716804223) if you're curious to read (short story: we don't think pre-event testing helps much).

One bit in your response I wanted to dig into a little: you wrote that `Put together, people start being contagious 4-5 days before the ideal time to take a test` and that `people are contagious before they reliably test positive.`. But that isn't totally clear to me from the articles you linked (or I may just be misreading!). An example case:

* Assume that median incubation period = 5 days, and median start of presymptomatic infectiousness is therefore ~3.5 days (24-48 hours beforehand)
* At T+4 days, only 33% of infected people will test positive (based on the papers @blanchardjeremy linked)
* However, only ~50% of infected people will be actively contagious at that time (because it is right at the median start of infectiousness)
* The question I can't find an answer to: how much does the 33% of infected people that will test positive _overlap_ with the 50% set of that are contagious (or most contagious) at that time?

If the answer to that last question is all of the people who test positive are also contagious, the situation isn't quite as bad as it initially sounds. Of the contagious people, 66% would get back positive results (33/50), even though there's technically a 67% false negative rate.

Finally - +1 to the "Ask MicroCOVID" section! We've found the FAQ to be super useful already :)

**Comment by beshaya** [comment]
I agree with your writeup to (1) - if you don't have reason to suspect someone was exposed (either a known exposure or exposure to a lot of random people), then timing a test is impossible. I know some colleges try to get around this by testing twice - on day 5 and 8 after students arrive. Without diving into numbers, I suspect that getting tested every 2-3 days leading up to an event would have value, but I just don't see anyone doing that.

I think you raise good points about how these timelines align. It seems totally plausible to me that there is correlation between the people who are contagious and the people who test positive (both rely on the virus coming out of your nose/throat, right?). In the absence of research that specifically says this overlap is the same, I'm a fairly wary of relying of relying on it. And even if the overlap were identical, you're still expecting 34% false negative among people who are actively contagious, so clearly the test is not a 1:1 map on contagiousness.

Another possible explanation is that the infection progresses at different paces in the upper and lower respiratory tract, and nasal swabs only check the upper respiratory tract.

Did my explanation of (2) make sense? I'll also note that it is getting easier and easier to get a large dose of microCOVIDs - for instance a schoolteacher in Denver would be exposed to [30,000 per day](https://www.microcovid.org/?distance=sixFt&duration=480&interaction=oneTime&personCount=10&riskProfile=average&setting=indoor&subLocation=US_08031&theirMask=basic&topLocation=US_08&voice=normal&yourMask=basic). As case levels rise, testing after an event becomes more valuable.

**Comment by cstigler** [comment]
Great thoughts on the potential overlap, and also the potential for nasal swabs to miss lower respiratory infections. I really wish there were more data on how test results relate to viral shedding, or (better yet) observed transmission rate. It's totally possible that they don't correlate at all, like you said - but it's also possible that the most contagious people test positive most reliably, and so testing is much more helpful than we assume. Would definitely be useful to know!

And yes, your explanation of (2) made sense and was very helpful! It makes sense intuitively that testing after high-risk activities should be more useful, because we can time it from the exposure. We've decided to go for a "mask/isolate for 5-7 days, get tested, unmask when test results come back" policy after high-risk events, which I think eliminates ~3/4 of the risk. Thanks again for the help as we work through this!
